Task_functions/index.js

```diff
@@ -1,11 +1,17 @@
-//Write code for the 1st task below this
-
-
-//Write your code above this line
-
-
-
-//Write code for the 2nd task below this
-
-
-//Write your code above this line
\ No newline at end of file
+//Write code for the 1st task below this
+

```

Comment:
The default value for the 'greeting' parameter should be set to 'Hello' instead of 'Hello'.